build_deploy.sh

```diff
@@ -0,0 +1,32 @@
+set -exv
```

Comment:
basically untested at this point, as with pr_checks looking for guidance on cleanly testing.